File: app/src/main/java/com/belov/artem/templateapp/presentation/view/activity/MainActivity.java
package com.belov.artem.templateapp.presentation.view.activity;


import android.os.Bundle;

import com.belov.artem.templateapp.R;

import com.belov.artem.templateapp.presentation.view.fragment.MainFragment;

import butterknife.ButterKnife;


public class MainActivity extends BaseActivity/* implements HasComponent*/{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
//        initializeInjector();

        addFragment(R.id.flContent, new MainFragment());
    }

}

File: app/src/main/java/com/belov/artem/templateapp/presentation/view/activity/MainActivity.kt
package com.belov.artem.templateapp.presentation.view.activity


import android.os.Bundle

import com.belov.artem.templateapp.R
import com.belov.artem.templateapp.presentation.view.fragment.FirstFragment

import butterknife.ButterKnife
import cyber.ru.activities.BaseActivity


class MainActivity : BaseActivity() {

    override val layoutResource: Int
        get() = R.layout.activity_main

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        ButterKnife.bind(this)
        addFragment(R.id.flContent, FirstFragment())
    }

}
